imjoy/server.py

```diff
@@ -262,6 +222,7 @@ async def add_cors_header(request: Request, call_next):
     return app
 
 
+# pylint: disable=too-many-arguments
```

Comment:
Keep this local inside the function where it's needed.

Reply:
I got it now, I moved into the function and it seems working.